libdnf/module/module_package.cpp

```diff
@@ -0,0 +1,280 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "libdnf/module/module_package.hpp"
+
+#include <modulemd-2.0/modulemd-module-stream.h>
+#include <modulemd-2.0/modulemd-profile.h>
+#include <modulemd-2.0/modulemd.h>
+
+#include <algorithm>
+#include <sstream>
+#include <string>
+#include <vector>
+
+namespace libdnf::module {
+
+
+const char * ModulePackage::get_name_cstr() const {
+    return modulemd_module_stream_get_module_name(md_stream);
+}
+
+
+std::string ModulePackage::get_name() const {
+    auto name = modulemd_module_stream_get_module_name(md_stream);
+    return name ? name : "";
```

Comment:
It is correct, but because this check is repeated om multiple lines, there can be a macro or inline. 

In package.cpp is
```
static inline std::string cstring2string(const char * input) {
    return input ? std::string(input) : std::string();
}
```
And it can be moved in private utils.

Reply:
There's `libdnf::utils::string::c_to_str()` already in `common/utils/string.hpp` (we should use it in `package.cpp`...)

Reply:
Thanks for pointing it out.